packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -129,3 +131,19 @@ export async function runWithRetry<T>(
 	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 	return result!;
 }
+
+const MaxReconnectDelayInMsWhenEndpointIsReachable = 30000;
```

Comment:
I suggest upping this value to 2 min. I have seen backend returning such value when under load. 30 seconds is very optimistic in my opinion.